float32 position
float32 position_confidence
